src/lambda/models/samLambdaRuntime.ts

```diff
@@ -6,7 +6,7 @@
 import { Runtime } from 'aws-sdk/clients/lambda'
 import { Map, Set } from 'immutable'
 
-export const nodeJsRuntimes: Set<Runtime> = Set<Runtime>(['nodejs8.10', 'nodejs10.x', 'nodejs12.x'])
+export const nodeJsRuntimes: Set<Runtime> = Set<Runtime>(['nodejs12.x', 'nodejs10.x', 'nodejs8.10'])
```

Comment:
Add a comment to denote that ordering matters here (see: https://github.com/aws/aws-toolkit-vscode/pull/1026/files#diff-14645e777a19d7090026b3ac7c4ac282R76 ). Does changing the order change how this outputs in the Create New Sam App flow?

Reply:
changed to:

```typescript
export function getDefaultRuntime(langId: string): string | undefined {
    switch (langId) {
        case 'typescript':
        case 'javascript':
            return nodeJsRuntimes.sort(compareSamLambdaRuntime).first()
        case 'csharp':
            return dotNetRuntimes.sort(compareSamLambdaRuntime).first()
        case 'python':
            return pythonRuntimes.sort(compareSamLambdaRuntime).first()
        default:
            return undefined
    }
}
```